lti_emailer/settings/base.py

```diff
@@ -141,8 +142,8 @@
 
 STATIC_URL = '/lti_emailer/static/'
 
-REDIS_HOST = SECURE_SETTINGS.get('redis_host', '127.0.0.1')
-REDIS_PORT = SECURE_SETTINGS.get('redis_port', 6379)
+REDIS_HOST = SECURE_SETTINGS.get('cache_host', '127.0.0.1')
```

Comment:
We currently have secure settings entries for redis_sessions_host and redis_sessions_port; @bermudezjd - what do you think about making these more general like redis_host and redis_port?